app/controllers/healthcare_controller.rb

```diff
@@ -27,11 +27,17 @@ def summary
     render 'summary/healthcare'
   end
 
+  def confirm
+    workflow = DocumentWorkflow.new(healthcare)
+    workflow.update_status(:complete)
```

Comment:
The front-end prevents showing the button that reaches this endpoint unless the section is in needs_review, and you can get there only when all questions are answered. 
`  - if healthcare.needs_review?
    = render partial: 'summary/confirm_and_save', locals: { path: confirm_healthcare_index_path }
`